lib/src/room.dart

```diff
@@ -167,6 +167,7 @@ class KurentoRoom extends ChangeNotifier implements Room {
       await room._init(session);
       return room;
     } catch (e) {
+      _log.shout('Unable to initialize the WebRTC Room.', e);
```

Comment:
This should help us greatly to debug issues in the future. Let's hope this is not too frequent.